src/visualization/types/Table/TableCell.tsx

```diff
@@ -44,7 +44,7 @@ const URL_REGEXP = /((http|https)?:\/\/[^\s]+)/g
 
 // NOTE: rip this out if you spend time any here as per:
 // https://stackoverflow.com/questions/1500260/detect-urls-in-text-with-javascript/1500501#1500501
-function asLink(str) {
+export function asLink(str) {
```

Comment:
Since this is an exported function, should we add a unit test here to validate this?

Reply:
Like:

```
expect(asLink('string').toEqual('string')
expect(asLink('http://www.google.com').toEqual(something else that isn't the string)
```

Reply:
👍   to unit tests

Reply:
Tests added.  And they caught an edge case so great call guys!  If there's any cases you think need further addition let me know.